enterprise/coderd/appearance_test.go

```diff
@@ -91,7 +92,7 @@ func TestCustomSupportLinks(t *testing.T) {
 	}
 	cfg := coderdtest.DeploymentConfig(t)
 	cfg.Support = new(codersdk.SupportConfig)
-	cfg.Support.Links = &codersdk.DeploymentConfigField[[]codersdk.LinkConfig]{
+	cfg.Support.Links = bigcli.Object[[]codersdk.LinkConfig]{
```

Comment:
I might call this `bigcli.Struct` instead of `Object`. `Object` sounds very Java or JavaScript-esque.